config/initializers/assets.rb

```diff
@@ -8,7 +8,7 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w(misc/*.js email.css ie*.css)
+Rails.application.config.assets.precompile += %w(app/controllers/*.js misc/*.js email.css ie*.css)
```

Comment:
Yeah let's keep all our JS in the `app/assets/javascripts` -- I expect only to find server-side code in `app/controllers`